diff --git a/node_modules/node-pty/binding.gyp b/node_modules/node-pty/binding.gyp
index 5f63978..b3309a0 100644
--- a/node_modules/node-pty/binding.gyp
+++ b/node_modules/node-pty/binding.gyp
@@ -5,9 +5,6 @@
     ],
     'conditions': [
       ['OS=="win"', {
-        'msvs_configuration_attributes': {
-          'SpectreMitigation': 'Spectre'
-        },
         'msvs_settings': {
             'VCCLCompilerTool': {
               'AdditionalOptions': [
diff --git a/node_modules/node-pty/deps/winpty/src/winpty.gyp b/node_modules/node-pty/deps/winpty/src/winpty.gyp
index 4abde70..9e49707 100644
--- a/node_modules/node-pty/deps/winpty/src/winpty.gyp
+++ b/node_modules/node-pty/deps/winpty/src/winpty.gyp
@@ -41,7 +41,6 @@
                 '-luser32',
             ],
             'msvs_configuration_attributes': {
-                'SpectreMitigation': 'Spectre'
             },
             'msvs_settings': {
                 # Specify this setting here to override a setting from somewhere
@@ -143,7 +142,6 @@
                 '-luser32',
             ],
             'msvs_configuration_attributes': {
-                'SpectreMitigation': 'Spectre'
             },
             'msvs_settings': {
                 # Specify this setting here to override a setting from somewhere
